src/NLog/Config/XmlLoggingConfiguration.cs

```diff
@@ -383,22 +364,12 @@ private void CheckUnusedTargets()
             InternalLogger.Debug("Unused target checking is completed. Total Rule Count: {0}, Total Target Count: {1}, Unused Target Count: {2}", this.LoggingRules.Count, configuredNamedTargets.Count, unusedCount);
         }
 
-        private void ConfigureFromFile(string fileName)
+        private void ConfigureFromFile(string fileName, bool autoReloadDefault = false)
```

Comment:
I prefer to make parameters of private methods always required (no optional). 

You could not know that, there is not coding rules (yet)